enterprise/metrics/src/main/java/org/neo4j/metrics/source/DBMetrics.java

```diff
@@ -195,6 +210,50 @@ public Long getValue()
             } );
         }
 
+        // check pointing
+        if ( config.get( MetricsSettings.neoCheckPointingEnabled ) )
```

Comment:
I have solved this problem in another PR: https://github.com/davidegrohmann/neo4j/commit/101b386926a419854808dd36538ac68777e8f3bd